### ISSUE TITLE
Need more section description languages

### ISSUE THREAD

**Issue by JacquesCarette** [JacquesCarette/Drasil#368]
We have languages for sections 1 (Reference Material), 2 (Introduction) and now references.  We still need to do the same for

- [x] Stakeholders
- [x] General System Description
- [x] Scope of the Project
- [x] Specific System Description
- [x] Requirements
- [x] Likely Changes
- [x] Traceability Matrices and Graphs
- [x] Values of Auxiliary Constants
- [x] Appendix

Once this is done `Verbatim` should be removed.

**Comment by niazim3** [comment]
As mentioned in the comments of commit 6da3116, I was wondering if subsections that are created by an overarching function should be their own types? For example, the current `stakeholderGeneral` function that is used to create the **Stakeholders** section automatically creates the subsections for "The Client" and "The Customer", which is why I'm hesitant in breaking up the `StkhldrProg` type... The same goes for the `scopeOfTheProjF` function that is used by `ScpOfProjSec`.  @JacquesCarette @szymczdm

**Comment by JacquesCarette** [comment]
The current `stakholderGeneral` has a very fixed view of how the document will look, and thus forces things to appear in subsections, and also forces subsections for the client and customer.

From an SE perspective, this is great, as it forces people to think about these things.  From a flexibility perspective, this is awful, as it forces a certain look.

My feeling is that the Drasil framework should, at its core, be flexible, and not force these things.  On top of that flexibility, we should layer some checks that will tell you if it is a "good SE document".  [These would, of course, have to be written].

Long story short: please do split.

**Comment by niazim3** [comment]
What is the status of this issue? It looks like it can be closed but I wanted to confirm that I'm not missing something important.

**Comment by JacquesCarette** [comment]
Well spotted - this is indeed done.

**Comment by niazim3** [comment]
@szymczdm 
In order to completely remove the use of `Verbatim`, the remaining sections need DocLang implementations: 

- [ ] Off-the-shelf solutions (used in GamePhysics)
- [ ] Properties of a Correct Solution (used in SWHS)

Related to #504.

**Comment by szymczdm** [comment]
Is that really all that's missing? Not bad.
